NOTICES.md

```diff
@@ -0,0 +1,12 @@
+# LMS-Toolkit
```

Comment:
@stephenfuqua should this be Ed-Fi-Actions?

Reply:
```suggestion
# Ed-Fi-Action Notices
```